tests/bundle/Functional/LocationTest.php

```diff
@@ -262,6 +264,27 @@ private function createUrlAlias(string $locationHref, string $urlAlias): string
 
         return $href;
     }
+
+    /**
+     * @covers \Ibexa\Rest\Server\Controller\Location::moveLocation
```

Comment:
Using method coverage declaration is highly discouraged by PHPUnit author. I'd suggest dropping it and/or using a class-level declaration.

see https://docs.phpunit.de/en/9.6/annotations.html#covers